app/helpers/assignment_index_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AssignmentIndexHelper
+  def cal_day(day:)
```

Comment:
It appears upon inspection that returning nil is okay here. It seems to me that having a time zone defined for the current day will come in handy when the functionality is changed to show hourly on call and isn't adversely affecting the method.
<img width="384" alt="screen shot 2017-07-28 at 11 52 32 am" src="https://user-images.githubusercontent.com/26842354/28725555-cd944d04-738b-11e7-989c-a229aa943f98.png">